examples/java/src/main/java/org/apache/beam/examples/WordCount.java

```diff
@@ -153,36 +153,23 @@ public String apply(KV<String, Long> input) {
    * <p>Inherits standard configuration options.
    */
   public interface WordCountOptions extends PipelineOptions {
+
+    /**
+     * By default, this example reads from a public dataset containing the text of
+     * King Lear. Set this option to choose a different input file or glob.
+     */
     @Description("Path of the file to read from")
     @Default.String("gs://apache-beam-samples/shakespeare/kinglear.txt")
     String getInputFile();
```

Comment:
Agree, but for good reason: `Output` is a prefix, not a file. Although technically the input can also be a glob, so perhaps leaving off `File` makes sense. I suppose `getInput` seems a bit vague.